Airbyte API 0.52.0 breaks provider tests

### Apache Airflow Provider(s)

airbyte

### Versions of Apache Airflow Providers

main

### Apache Airflow version

main

### Operating System

Linux

### Deployment

Virtualenv installation

### Deployment details

Breeze tests

### What happened

Tests on main as well as PRs re-testing airbyte provider fail in pytests with:

```
(...)
airflow/providers/airbyte/hooks/airbyte.py:56: in __init__
    self.airbyte_api = self.create_api_session()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <airflow.providers.airbyte.hooks.airbyte.AirbyteHook object at 0x7f1847b62060>

    def create_api_session(self) -> AirbyteAPI:
        """Create Airbyte API session."""
>       credentials = SchemeClientCredentials(
            client_id=self.conn["client_id"],
            client_secret=self.conn["client_secret"],
            TOKEN_URL=self.conn["token_url"],
        )
E       TypeError: SchemeClientCredentials.__init__() got an unexpected keyword argument 'TOKEN_URL'

airflow/providers/airbyte/hooks/airbyte.py:71: TypeError
```

...if provider yaml in airflow/providers/airbyte/provider.yaml since airbyte-api 0.52.0 was released.

### What you think should happen instead

_No response_

### How to reproduce

run `breeze testing tests --test-type '[Providers[airbyte]'`

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

--- a/airflow/providers/airbyte/hooks/airbyte.py
+++ b/airflow/providers/airbyte/hooks/airbyte.py
@@ -71,7 +71,7 @@ class AirbyteHook(BaseHook):
         credentials = SchemeClientCredentials(
             client_id=self.conn["client_id"],
             client_secret=self.conn["client_secret"],
-            TOKEN_URL=self.conn["token_url"],
+            token_url=self.conn["token_url"],
         )
 
         return AirbyteAPI(

--- a/airflow/providers/airbyte/provider.yaml
+++ b/airflow/providers/airbyte/provider.yaml
@@ -51,7 +51,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.8.0
-  - airbyte-api==0.51.0  # v0.52.0 breaks hooks, see https://github.com/apache/airflow/issues/42154
+  - airbyte-api>=0.52.0
 
 integrations:
   - integration-name: Airbyte

--- a/generated/provider_dependencies.json
+++ b/generated/provider_dependencies.json
@@ -1,7 +1,7 @@
 {
   "airbyte": {
     "deps": [
-      "airbyte-api==0.51.0",
+      "airbyte-api>=0.52.0",
       "apache-airflow>=2.8.0"
     ],
     "devel-deps": [],

